Index: index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>var express = require('express');\r\nvar app = express();\r\n\r\napp.use(express.static('static'));\r\nconsole.log(\"helo\");\r\nrequire('./routes/outside')(app);\r\nvar server = app.listen(3000, function () {\r\n    console.log(\"On: 3000\");\r\n});
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.js b/index.js
--- a/index.js	(revision 78c04116102839c0b2919448290fc2d883c38764)
+++ b/index.js	(date 1616322113106)
@@ -2,7 +2,7 @@
 var app = express();
 
 app.use(express.static('static'));
-console.log("helo");
+console.log("kek");
 require('./routes/outside')(app);
 var server = app.listen(3000, function () {
     console.log("On: 3000");
